node/core/dispute-coordinator/src/real/metrics.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use polkadot_node_subsystem_util::metrics::{self, prometheus};
+
+#[derive(Clone)]
+struct MetricsInner {
+	/// Rolling counter value of currently open disputes.
+	open: prometheus::Gauge<prometheus::U64>,
```

Comment:
nit: instead of having one gauge it might be better to have 2 counters as suggested here: https://github.com/paritytech/polkadot/pull/1536/#discussion_r469238780
esp. considering that prometheus collects metrics once in a minute

Reply:
:+1:  Sampling theorem strikes again.